apps/orams/components/HomePage/HomePage.js

```diff
@@ -1,6 +1,26 @@
 import React from 'react'
+import styles from './HomePage.scss'
 
-const HomePage = () => <div />
+const HomePage = () =>
+  <div>
+    <section className={styles.heroMain}>
+      <div className={styles.wrapper}>
+        <div className={styles.contentHero}>
+          <h1 className={styles.siteTitle}>
+            ORAMS Marketplace<span className={styles.badgeBeta}>BETA</span>
+          </h1>
+          <p className="tagline">Where government buyers and digital specialists do business.</p>
+        </div>
+        <div className={styles.calloutHero}>
+          <h2>
+            <a href="/become-a-seller">Become a seller</a>
+          </h2>
+          <p>Access more digital opportunities across government.</p>
+        </div>
+      </div>
+    </section>
+    <section className={styles.contentMain} />
+  </div>
 
 HomePage.defaultProps = {}
```

Comment:
if there are no props, do we need to worry about props?

Reply:
Fixed